### PR TITLE
initial 17.1 enablement

### PR DESCRIPTION
**Pull request by stuggi**
also adds help output to Makefile 

Depends-On: https://github.com/openstack-k8s-operators/osp-director-operator/pull/796

### CODE REVIEW

**Comment by stuggi** [comment]
blocked by https://issues.redhat.com/browse/OSPK8-681

**Review by abays — COMMENTED** [review]
+1 from me

File: ansible/vars/17.1.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+csv_version: latest-17.1
+osp_release_auto_version: 17.1-RHEL-9
+osp_release_auto_compose: passed_phase1
+
+osp_release_defaults:
+  base_image_url: http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/9.2/20230414.43/images/rhel-guest-image-9.2-20230414.43.x86_64.qcow2
+
+# (mschuppert) can be deleted when released
+openstackclient_image: "rhos-qe-mirror-rdu2.usersys.redhat.com:5002/rh-osbs/rhosp17-openstack-tripleoclient:{{ osp_release_auto.tag }}"
```
Comment (olliewalsh) [inline ansible/vars/17.1.yaml:10]: hrmm, why did I not use `{{ registry_proxy }}` for these images? I probably just missed them. Will test and fix all of the vars files in a follow-up patch if it works

**Review by olliewalsh — APPROVED** [review]
/lgtm

**Pull request merged**